alacritty/src/renderer/mod.rs

```diff
@@ -359,30 +360,46 @@ impl GlyphCache {
     }
 }
 
+// XXX these flags must be in sync with their usage in text.*.glsl shaders.
+bitflags! {
+    #[repr(C)]
+    struct RenderingGlyphFlags: u8 {
+        const WIDE_CHAR = 0b0000_0001;
+        const COLORED   = 0b0000_0010;
+    }
+}
+
 #[derive(Debug)]
 #[repr(C)]
 struct InstanceData {
     // Coords.
     col: u16,
     row: u16,
+
     // Glyph offset.
     left: i16,
     top: i16,
+
     // Glyph size.
     width: i16,
     height: i16,
+
     // UV offset.
     uv_left: f32,
     uv_bot: f32,
+
     // uv scale.
     uv_width: f32,
     uv_height: f32,
+
     // Color.
     r: u8,
     g: u8,
     b: u8,
-    // Flag indicating that a glyph uses multiple colors; like an Emoji.
-    multicolor: u8,
+
+    // Cell flags.
```

Comment:
This comment is useless, if we want to add a comment here, we should provide additional value over just the variable name. An example for what is stored in the flags at the moment could be helpful (Cell flags like multicolor or fullwidth character). Otherwise we might as well just remove it.

Reply:
I've just followed other useless comments here.